index.js

```diff
@@ -262,6 +263,9 @@ class Autohook extends EventEmitter {
     
     if (!webhookUrl) {
       this.startServer();
+      if(this.auth.ngrok_secret){
+        await ngrok.authtoken(token)
```

Comment:
It looks like ngrok.authtoken is taking `token` (the user's access token). Should this be `ngrok_secret` or `this.ngrokSecret` instead?

Reply:
Yes, it was indeed. Thanks. I will correct it